README.md

```diff
@@ -1,19 +1,18 @@
 ![IMAPBOX](logo.png)
 
-
-
-Dump imap inbox to a local folder in a regular backupable format: html, json and attachements.
+Dump IMAP inbox to a local folder in a regular backupable format: HTML, JSON and attachments.
 
 This program aims to save a mailbox for archive using files in indexable or searchable formats.
-The produced files should be readables without external software, for example, to find an email in backups using only the terminal
+The produced files should be readable without external software, for example, to find an email in backups using only the terminal.
 
-For each email in the imap mailbox, create a folder with the following content:
+For each email in the IMAP mailbox, create a folder with the following content:
 
-* __message.html__ if an html part exists for the message body. the message.html will allways be in utf-8, the embeded images links are modified to refer to the attachments subfolder
-* __attachements__ The attachements folder contains the attached files and the embeded images
-* __message.txt__ this file contain the body text if available in the original email, allways converted in utf-8
-* __metadata.json__ Various informations in JSON format, date, recipients, body text, etc... This file can be used from external applications or a search engine like [elasticsearch](http://www.elasticsearch.com/)
-* __raw.eml.gz__ A gziped version of the email in eml format
+* __message.html__ - If an html part exists for the message body. the `message.html` will always be in UTF-8, the embedded images links are modified to refer to the attachments subfolder.
+* __message.pdf__ - This file is created from `message.html` when the `wkhtmltopdf` option is set in the config file.
+* __attachments__ - The attachments folder contains the attached files and the embeded images.
+* __message.txt__ - This file contain the body text if available in the original email, always converted in UTF-8.
+* __metadata.json__ - Various informations in JSON format, date, recipients, body text, etc... This file can be used from external applications or a search engine like [elasticsearch](http://www.elasticsearch.com/).
+* __raw.eml.gz__ - A gziped version of the email in `.eml` format.
 
 Imapbox was designed to archive multiple mailboxes in one common directory tree,
 copies of the same message spread knew several account will be archived once using the Message-Id property.
```

Comment:
the chapter "Install" needs to be improved or maybe you can improve something for the pdfkit dependency.
I had to install pdfkit on my system to make the script work, but have added nothing in my config file. The best thing would be to make the script work without pdfkit if the config file does not contain anything about pdfkit. If this is not possible, we must add pdfkit as an install dependency.

Reply:
I just used `apt install python-pdfkit` and the script worked as before (without pdf) on my existing test configuration

Reply:
right ! i myself have pretty much followed this guide: https://pypi.python.org/pypi/pdfkit
i could link that and write a sentence or two in the `README`.

> The best thing would be to make the script work without pdfkit if the config file does not contain anything about pdfkit.

you're right ! 

could we simply put `import pdfkit` within the `createPdfFile(..)` function ? – since it's only called when the `wkhtmltopdf` option is available.

Reply:
> could we simply put `import pdfkit` within the `createPdfFile(..)` function ?

on second thought, this seems like bad practice, since import would then be called on every function call.

Reply:
@polo2ro how about the following (found this on [stackoverflow](https://stackoverflow.com/a/14050282)):

```py
import imp
try:
    imp.find_module('pdfkit')
    has_pdfkit = True
except ImportError:
    has_pdfkit = False
```

or 

```py
import pkgutil
pdfkit_loader = pkgutil.find_loader('pdfkit')
has_pdfkit = pdfkit_loader is not None
```

which one would you prefer more ?

Reply:
the one with pkgutil because this will be similar in python3